### PR TITLE
Feature/backup status

### PR DESCRIPTION
**Pull request by louisfelix**
Adding recovery_services_vault_backup_collector

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -71,6 +72,7 @@ func main() {
 	prometheus.MustRegister(serviceFabricClustersCollector)
 	prometheus.MustRegister(eventGridTopicsCollector)
 	prometheus.MustRegister(trafficManagerProfilesCollector)
+	prometheus.MustRegister(recoveryServicesBackupCollector)
```
Comment (rolandvarga) [inline main.go]: With the addition of the newest collector this is starting to get large in size. I would recommend enumerating [(iota)](https://yourbasic.org/golang/iota/) the different collector types, and register by looping over them.

File: go.mod [hunk 2]
```diff
@@ -4,9 +4,10 @@ go 1.12
 
 require (
 	contrib.go.opencensus.io/exporter/ocagent v0.6.0 // indirect
-	github.com/Azure/azure-sdk-for-go v36.2.0+incompatible
+	github.com/Azure/azure-sdk-for-go v39.3.0+incompatible
```
Comment (rolandvarga) [inline go.mod]: Does this change have any impact on the other collectors introduced by the application?

Comment (louisfelix) [reply go.mod]: No, tests are running fine!

File: main.go (hunk 1 above)
Comment (louisfelix) [reply main.go]: I did not use iota, but I enhanced registration by looping!

**Pull request merged**